bundles/org.openhab.ui/web/src/pages/addons/addons-store.vue

```diff
@@ -95,10 +119,11 @@
           @addonButtonClick="addonButtonClick"
           :addons="allAddons.filter((a) => a.type === 'automation' && a.contentType === 'application/vnd.openhab.uicomponent;type=blocks')"
           :install-action-text="'Add'"
-          :title="'Block Libraries'"
+          :title="'Blockly Libraries'"
```

Comment:
These are called "Block Libraries" and have been from the start.
Sorry but you don't get to rename them on a whim.

Reply:
#2404

Reply:
Thanks for the catch and the fix!